nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListenSyslog.java

```diff
@@ -152,7 +145,7 @@
                 "The maximum number of Syslog events to add to a single FlowFile. If multiple events are available, they will be concatenated along with "
                 + "the <Message Delimiter> up to this configured maximum number of messages")
         .addValidator(StandardValidators.POSITIVE_INTEGER_VALIDATOR)
-        .expressionLanguageSupported(false)
+        .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
```

Comment:
If you enable EL, you'll need to call evaluateAttributeExpressions when you use the property below.  Also, I'm not sure FLOWFILE_ATTRIBUTES is appropriate here, since ListenSyslog is annotated with INPUT_FORBIDDEN.

Reply:
Good catch, that should not have been changed, will revert back.

Reply:
It might still make sense to let it be ExpressionLanguage.VARIABLE_REGISTRY if you want to allow for the additional flexibility, though it seems like a setting that wouldn't likely be externalized to the variable registry.

Reply:
Yes, that would be an option, but agree that there doesn't seem to be much value in making it configurable through the variable registry.